.devcontainer/setup.sh

```diff
@@ -11,3 +11,7 @@ micromamba env create -f environment.yml --yes
 # user (same applies to `conda activate`)
 
 git submodule update --init
+
+# Enables users to activate environment without having to specify the full path
+echo "envs_dirs:
+  - /home/codespace/micromamba/envs" > /opt/conda/.condarc
```

Comment:
There will be a line break here, is that OK?

Reply:
Yes, this is how it's supposed to look in the final file - see https://conda.io/projects/conda/en/latest/user-guide/configuration/use-condarc.html#specify-env-directories